### DISCUSSION TITLE
Add Facebook instant game export support

### DISCUSSION THREAD

**Discussion by ghost** [godotengine/godot-proposals#4120] [Engine Core]
Facebook instant game need the Facebook instant game sdk. So please add this sdk. Without the sdk we can't upload our html games

**Comment by theraot** [comment]
I don't think something like this should be in core. In general these integrations should be plugins. On one hand, not every game and developer will want the integration, and for them it is making the engine and game needlessly heavier. On the other, I'd prefer Godot do not play favorites with third parties.

That SDK is web, right? So we are not talking about integrating a native library, which might require GDNative/GDExtension or compile Godot with a module. Instead, I'd expect it to be JavaScript. So, could you not use it form the [JavaScript](https://docs.godotengine.org/en/stable/classes/class_javascript.html) class?

**Reply by ghost** [reply]
I haven't tried. Because I haven't tried JavaScript. 


if it could be possible then Can you make the plugin?

**Comment by Calinou** [comment]
There's already a proposal tracking this: https://github.com/godotengine/godot-proposals/issues/2695

**Comment by ghost** [comment]
> There's already a proposal tracking this: https://github.com/godotengine/godot-proposals/issues/2695

But they are not working on it. The proposal was maid 10 month ago. But no improvement

**Reply by Calinou** [reply]
It's likely not technically feasible (see https://github.com/godotengine/godot-proposals/discussions/4120#discussioncomment-2267008).

**Comment by eon-s** [comment]
The last time I've worked on a FIG project (like a year ago or so), the platform didn't supported webassembly, so it will be impossible if they keep those restrictions.
